### DISCUSSION TITLE
Please add the ability to remove shortcut arrows.

### DISCUSSION THREAD

**Discussion by agoodeill** [valinet/ExplorerPatcher#3558] [Ideas]
Adding a feature to remove the arrow on shortcuts, making the icons cleaner.

**Comment by pyrates999** [comment]
https://answers.microsoft.com/en-us/windows/forum/all/how-do-you-remove-the-desktop-icons-shortcut/bfff8688-1313-44a7-a86f-5015c44eaa53
